packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -1431,9 +1431,12 @@ def call_next_link_request_builder(self, builder: PagingOperationType) -> List[s
                     "})",
                 ]
             )
-            retval.append(
-                f'_next_request_params["api-version"] = {api_version_param.full_client_name}'
+            api_version = (
+                "self._api_version"
+                if self.code_model.options["multiapi"] and builder.group_name
```

Comment:
It is by design that check `builder.group_name` here since mixin operation is tied with client which has no attribute `_api_version`: https://github.com/Azure/autorest.python/pull/2452/commits/f8d50d536582c6ddedacbd64761aa2117b74d5e8#diff-2e5002d81b5abd3b275758f1f3d04a35e76d6c1eda5c776702a097fc85ffd9c7